docs/tab-widgets/jaeger-sampling.asciidoc

```diff
@@ -0,0 +1,14 @@
+// tag::ess[]
+Visit the {kibana-ref}/agent-configuration.html[Agent configuration] page in the APM app to add a new sampling rate.
+
+// end::ess[]
```

Comment:
Does central config sampling in ESS JustWork™️? I changed the sampling rate but didn't see anything in the Jaeger client logs. Is there an easy way to determine if the sampling rate is applied? I also tried looking at `transaction.sampled`, but couldn't spot an obvious difference.

Reply:
It should. Were you testing with HotROD? It disables remote sampling by default: https://github.com/jaegertracing/jaeger/blob/c0fb781662eacb4ec6f544e78d081528541c2473/examples/hotrod/pkg/tracing/init.go#L38-L39

(The Jaeger client default is remote: https://github.com/jaegertracing/jaeger-client-go/blob/398f7a767754a7dd438d6ff2f8631698bf913696/config/config.go#L238-L243, but this app overrides it)

Reply:
Aha! That's what it was. Thanks for the clarification.